src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/ExecutionServiceCluster.scala

```diff
@@ -0,0 +1,35 @@
+package org.broadinstitute.dsde.rawls.dataaccess
+
+import org.broadinstitute.dsde.rawls.RawlsException
+import org.broadinstitute.dsde.rawls.dataaccess.slick.WorkflowRecord
+import org.broadinstitute.dsde.rawls.model._
+
+import scala.concurrent.Future
+import scala.util.Try
+
+/**
+  * Created by davidan on 6/14/16.
+  */
+// TODO: keys for the map of members should probably be strings
+trait ExecutionServiceCluster extends ErrorReportable {
+  override val errorReportSource = "cromwell"
+
+  // ====================
+  // facade methods
+  // ====================
+  def submitWorkflow(wdl: String, inputs: String, options: Option[String], userInfo: UserInfo): Future[ExecutionServiceStatus]
+
+  def submitWorkflows[T](wdl: String, inputs: Seq[String], options: Option[String], userInfo: UserInfo)(wf: T): Future[Seq[Either[ExecutionServiceStatus, ExecutionServiceFailure]]]
+
+  def status[T](id: String, userInfo: UserInfo)(wf: T): Future[ExecutionServiceStatus]
```

Comment:
I would have expected `status(id: String, execServiceId: String, userInfo: UserInfo)` here. Why isn't it that? We're going to have to add some notion of `execServiceId` to the workflow anyway; it seems reasonable to me that if the caller has to provide a workflow ID, they should also provide which cromwell it ran on.